.eslintrc

```diff
@@ -3,9 +3,11 @@
   "extends": [
     "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended",
+    "plugin:react/recommended", // Specify rules for React
```

Comment:
@daniel-ivanco any specific reason you added this? 

Recommended rules are already part of next plugin. We don't need anything else.